data/json/mapgen/laundromat.json

```diff
@@ -0,0 +1,261 @@
+      ["tshirt", 70],
+      ["longshirt", 80],
+      ["polo_shirt", 65],
+      ["dress_shirt", 60],
+      ["tank_top", 50],
+      ["camisole", 30],
+      ["bra", 30],
+      ["undershirt", 30],
+      ["boxer_shorts", 30],
+      ["briefs", 15],
+      ["panties", 30],
+      ["boy_shorts", 25],
+      ["sweatshirt", 75],
+      ["sweater", 75],
+      ["hoodie", 65],
+      ["jacket_light", 50],
+      ["jacket_jean", 35],
+      ["blazer", 35],
+      ["jacket_leather", 30],
+      ["coat_winter", 50],
+      ["peacoat", 30],
+      ["gloves_light", 35],
+      ["mittens", 30],
+      ["gloves_wool", 33],
+      ["gloves_winter", 40],
+      ["gloves_liner", 25],
+      ["gloves_leather", 45],
+      ["scarf", 45],
+      ["knit_scarf", 35],
+      ["long_knit_scarf", 5],
+      ["scarf_long", 1],
+      ["hat_cotton", 45],
+      ["hat_knit", 25],
+      ["hat_newsboy", 20],
+      ["hat_fur", 15],
+      ["under_armor", 20],
+      ["under_armor_shorts", 20],
+      ["tights", 20],
+      ["leggings", 20],
+      ["stockings", 20],
+      ["balclava", 15],
+      ["pants_ski", 60],
+      ["long_underpants", 40],
+      ["long_undertop", 40],
+      ["union_suit", 20],
+      ["arm_warmers", 20],
+      ["leg_warmers", 20],
+      ["trenchcoat_leather", 12],
+      ["sleeveless_trenchcoat", 2],
+      ["sleeveless_trenchcoat_leather", 2],
+      ["sleeveless_trenchcoat_fur", 1],
+      ["duster_leather", 12],
+      ["sleeveless_duster", 2],
+      ["sleeveless_duster_leather", 2],
+      ["sleeveless_duster_fur", 1],
+      ["cloak", 5],
+      ["house_coat", 25],
+      ["flotation_vest", 1],
+      ["fishing_waders", 5],
+      ["wetsuit", 5],
+      ["wetsuit_spring", 5],
+      ["wetsuit_gloves", 10],
+      ["wetsuit_booties", 10],
+      ["wetsuit_hood", 5],
+      ["apron_leather", 1],
+      ["clown_suit", 1],
+      ["clownshoes", 1],
+      ["bondage_suit", 1],
+      ["corset", 10],
+      ["pants_army", 10],
+      ["jacket_army", 10],
+      ["winter_pants_army", 10],
+      ["winter_jacket_army", 10],
+      ["winter_gloves_army", 10],
+      ["army_top", 20],
+      ["tux", 1],
+      ["dress_wedding", 1],
+      ["porkpie", 5],
+      ["skinny_tie", 5],
+      ["tieclip", 2],
+      ["collarpin", 2],
+      ["jersey", 40],
+      ["postman_shorts", 5],
+      ["postman_shirt", 5],
+      ["postman_hat", 5],
+      ["maid_dress", 3],
+      ["maid_hat", 3],
+      ["halter_top", 50],
+      ["linuxtshirt", 20],
+      ["kilt", 5],
+      ["nanoskirt", 10],
+      ["sleeveless_tunic", 5],
+      ["ear_spool", 30],
+      ["fedora", 5],
+      ["straw_hat", 5],
+      ["straw_fedora", 5],
+      ["kufi", 2],
+      ["kippah", 2],
+      ["thawb", 1],
+      ["kittel", 1],
+      ["cassock", 1],
+      ["robe", 5],
+      ["eboshi", 1],
+      ["kariginu", 1],
+      ["kimono", 3]
+    ]
+},
+
+{
+      "type"  : "item_group",
+   "id"    : "laundromat_bleach",
+   "items" : [
+    ["bleach",        50]
```

Comment:
A lack of either a charges-min or count-min here is presumably the cause of massive amounts of bleach.